atom/browser/browser.h

```diff
@@ -261,6 +261,9 @@ class Browser : public WindowListObserver {
   bool is_ready() const { return is_ready_; }
   const util::Promise& WhenReady(v8::Isolate* isolate);
 
+  bool secure_mode_enabled() const { return secure_mode_enabled_; }
+  void EnableSecureMode(mate::Arguments* args);
```

Comment:
`atom::Browser` shouldn't receive `args` IMO.  The arg parsing should be done in `atom::api::App` and this should take no arguments